simulation/agents.py

```diff
@@ -85,6 +84,10 @@ def get_weibull_value_from_percentile(
         epsilon = 0.0000001
         return beta * (-math.log(1 + epsilon - percentile)) ** (1 / alpha)
 
+    @staticmethod
+    def evaluate_trial(p: float) -> bool:
```

Comment:
I do get the name here, but I think that `true_with_probability(p: float)` is easier to read?